rllib/core/models/tests/test_base_models.py

```diff
@@ -266,6 +272,43 @@ def compile_me(input_dict):
         break_reasons_list = dynamo_explanation[4]
         self.assertEquals(len(break_reasons_list), 1)
 
+    @unittest.skipIf(not _dynamo_is_available(), "torch._dynamo not available")
+    def test_torch_compile_forwards(self):
+        """Test if logic around TorchCompileConfig works as intended."""
+
+        spec = SingleAgentRLModuleSpec(
+            module_class=PPOTorchRLModule,
+            observation_space=gym.spaces.Box(low=0, high=1, shape=(32,)),
+            action_space=gym.spaces.Box(low=0, high=1, shape=(1,)),
+            model_config_dict={},
+            catalog_class=PPOCatalog,
+        )
+        torch_module = spec.build()
+
+        compile_config = TorchCompileConfig(compile_forward_train=True)
+
+        compile_config.compile(torch_module)
+
+        # We should still be able to call the forward method
+        torch_module._forward_train({"obs": torch.randn(1, 32)})
+        self.assertTrue(hasattr(torch_module, "_compiled_forward_train"))
+
+        # Compile again with different config and see if everything still works
+
+        compile_config = TorchCompileConfig(
+            compile_forward_train=True,
+            compile_forward_inference=True,
+            compile_forward_exploration=True,
+        )
+
+        compile_config.compile(torch_module)
```

Comment:
We are still doing `config.compile(torch_module)`

Reply:
Wait we are not checking this test on CI? because it's skipped? is there any way to check this now when we haven't moved the CI to torch 2.0?